model-archiver/model_archiver/arg_parser.py

```diff
@@ -96,4 +96,13 @@ def export_model_args_parser():
                                         'name as that provided in --model-name in the path specified by --export-path\n'
                                         'will overwritten')
 
+        parser_export.add_argument('-c', '--convert',
```

Comment:
This arg name does not seem intuitive enough, though reading the description, this appears to be a very custom requirement to skip or process onnx models.

Reply:
I am open for opinions on what this should be :)